online-inference/tensorizer-isvc/gptj_6b_hf/flask/transformer_flask.py

```diff
@@ -0,0 +1,68 @@
+from typing import List, Dict
+import logging
+import requests
+import numpy as np
+import base64
+import time
+import torch
+from flask import Flask
+from transformers import AutoTokenizer, GPTJForCausalLM
+
+# The signature name is defined at time of export, in signature_def_map supplied to builder
+# Tensorflows default is serving_default
```

Comment:
Is TensorFlow used in this example, or is this a relic of some previous revision?